### PR TITLE
idb adaptation

### PR DESCRIPTION
**Pull request by AlexeyBudynkov**
*calabash FBSimulatorControl repository was changed to the original idb repository: https://github.com/facebook/idb/
*CocoaLumberJack framework was moved from calabash FBSimulatorControl to this repository
*for functionality that was removed from idb repository but which is still required it was created an additional interface - FBLegacy. 
Also for the functions related to xcappdata containers downloading and uploading there was PrivateHeaders directory imported and added some Xcode legacy libraries preload 
*some tests and build configurations were fixed for iOS 15 and Xcode 13.0 build

### CODE REVIEW

File: iOSDeviceManager_test [hunk 1]
```diff
@@ -0,0 +1,13 @@
+bundle install
```
Comment (IlyaBausovAkvelon) [inline iOSDeviceManager_test]: Is it supposed to be here?

File: iOSDeviceManager/IDMVersionDefines.h [hunk 2]
```diff
@@ -1,11 +1,6 @@
...
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT license.
-
 /*
  Do Not Manually Edit This File
 */
-
-#define IDM_GIT_SHORT_REVISION @"94c9a5f"
-#define IDM_GIT_BRANCH @"update-cli"
+#define IDM_GIT_SHORT_REVISION @"679df5a"
```
Comment (IlyaBausovAkvelon) [inline iOSDeviceManager/IDMVersionDefines.h]: Should it be removed?

**Review by IlyaBausovAkvelon — APPROVED** [review]

File: iOSDeviceManager_test (hunk 1 above)
Comment (AlexeyBudynkov) [reply iOSDeviceManager_test]: I created this file as a hint. I think it could be removed.

File: iOSDeviceManager/IDMVersionDefines.h (hunk 2 above)
Comment (AlexeyBudynkov) [reply iOSDeviceManager/IDMVersionDefines.h]: Actually, I don't know. It was set automatically during the application building.

**Pull request merged**